### PR TITLE
Fix further pylint issues

### PR DESCRIPTION
**Pull request by kasium**

### CODE REVIEW

**Comment by Bachmann1234** [comment]
thanks!

File: diff_cover/violationsreporters/base.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def violations(self, src_path):
         Return a list of Violations recorded in `src_path`.
         """
 
+    @abstractmethod
```
Comment (barrywhart) [inline diff_cover/violationsreporters/base.py:37]: What was the intent behind marking this method as `@abstractmethod`? This is causing SQLFluff's implementation of this plugin to break, with the error message:
```
TypeError: Can't instantiate abstract class SQLFluffViolationReporter with abstract methods measured_lines
```

What should we do to get SQLFluff working again?

Comment (kasium) [reply diff_cover/violationsreporters/base.py:37]: The idea was that all Reports are expected to implement this method, therefore I marked it as abstract. To let SQLFluff work again just implement the method and return None as the original

However, in my understanding the internals of diff-cover beside the pluggy plugin and the cli are not really stable. @Bachmann1234  maybe it's worth to decide upon that

Comment (Bachmann1234) [reply diff_cover/violationsreporters/base.py:37]: so Ill be the first to admit the plugin stuff was a half baked idea for me. 

Back when I did it the idea was to let me move the implementations out of diff_cover and into other repos so we did not have to know so much about all these tools inside _this_ repo. 

WE broke this by accident. Ill look into undoing this part of it and getting a new release out later today

Comment (barrywhart) [reply diff_cover/violationsreporters/base.py:37]: The plug-in stuff has been pretty useful for SQLFluff, I think. We have already fixed this breakage on the SQLFluff side, so no real rush there -- feel free to take the time to think through what seems right for the long term (or to do nothing).

I think I was the person who implemented the diff-cover plug-in support a year or two ago. 😛

Comment (Bachmann1234) [reply diff_cover/violationsreporters/base.py:37]: oh well I fixed it in 6.1.1 that just went out.

Also! Im sorry I erased you from the history just now. I think my memory combined a few events...

yes! You did implement the plugin system with this PR https://github.com/Bachmann1234/diff_cover/pull/128 

 I had started with this PR https://github.com/Bachmann1234/diff_cover/pull/30 which was some refactoring to try and prepare the system for plugins. However, I never actually went ahead and did it. 

So once again @barrywhart sorry about that! Ill try to be a bit more careful when talking about my contributions vs others!

Comment (barrywhart) [reply diff_cover/violationsreporters/base.py:37]: No problem! I mentioned the history mainly to share blame for anything half baked. 😄 I'd be happy to help if you see a need for additional work in this area.

Thanks for the quick turnaround.

**Pull request merged**